app/views/katello/api/v2/subscriptions/base.json.rabl

```diff
@@ -13,3 +13,12 @@ attributes :sockets, :cores, :ram
 attributes :instance_multiplier, :stacking_id, :multi_entitlement
 attributes :type
 attributes :name => :product_name
+attributes :unmapped_guest
+attributes :virt_only
```

Comment:
Way to turn my question on me :) I was more looking for your opinion -- my question being one of are these attributes that make sense in most use cases or a very select few of cases?